tests/typecheck/models/test_create.yml

```diff
@@ -60,6 +60,27 @@
                 class Child4(Child1):
                     value4 = models.IntegerField()
 
+-   case: mro_is_walked_for_field_annotations
+    main: |
+        from myapp.models import Child
+        Child.objects.create(dct={"hello": "world"})  # no errors
```

Comment:
Can you please add:

```yml
c: Child
reveal_type(c.dct)
```

?

This is needed to prevent a possible regression that `dct` is `Any`.

Reply:
I don't think that's relevant / useful -- the type for `dct` is explicitly annotated.  the bug here is strictly in the plugin parts of `create`